### DISCUSSION TITLE
Problem accessing account with 2-factor authentication

### DISCUSSION THREAD

**Discussion by LaryssaGomes** [community/community#46545] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

I have an account that has 2-factor authentication, I had the PC corrupted last Friday, so I asked for the authentication codes that were on it. I have a place where I write down my passwords and the github password that is written down does not work at login, and when I go to "forgot password" I am taken to 2-factor authentication. So I can't access the account, could anyone help with that?

**Comment by SilasRodrigues19** [comment]
Oi @LaryssaGomes 

Você pode acessar esse link aqui com instruções de recuperação de conta e tentar recuperar:

https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa/recovering-your-account-if-you-lose-your-2fa-credentials

Caso não resolva, uma outra alternativa seria enviar uma solicitação para o Suporte do GitHub, através desse link:

https://support.github.com/contact/account

**Reply by LaryssaGomes** [reply]
obrigada @SilasRodrigues19 entrarei em contato com o suporte

**Reply by SilasRodrigues19** [reply]
Acho que vai demorar pra eles responderem mas boa sorte
